benchmarking/requirements.txt

```diff
@@ -1,6 +1,6 @@
+anonlink-client==0.0.1
```

Comment:
This will be 0.1.0 pretty soon

Reply:
Sweet. Our depandabot will pester us when it is released :-)